### ISSUE TITLE
Cannot clear 'select' properties

### ISSUE THREAD

**Issue by Adjective-Object** [jamalex/notion-py#15]
trying to write 'None' to a select properties fails with the following error

```
  File "/home/adjective/.local/share/virtualenvs/notion-strip-7FiIVxN3/src/notion/notion/collection.py", line 389, in _convert_python_to_notion
    val = val.split(",")[0]
AttributeError: 'NoneType' object has no attribute 'split'
```

https://github.com/jamalex/notion-py/blob/master/notion/collection.py#L380

**Comment by jamalex** [comment]
Thanks for reporting! Fixed in https://github.com/jamalex/notion-py/pull/22, and will be released soon in 0.0.21

**Comment by jamalex** [comment]
Version 0.0.21 has now been [released to PyPI](https://pypi.org/project/notion/0.0.21/)! Please give it a try and see how it works for you. Thanks for reporting issues!
